docs/development/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _developer-docs:
+
+************
+Contributing
+************
+
+The contributor documentation contains instructions for how to contribute to ``astropy`` or
+affiliated packages. This includes setting up a development environment, installing and
+testing the development version, as well as coding, documentation, and testing
+guidelines.
+
+For newcomers the process may initially seem overwhelming, but with a little patience
+and practice you will see that it is not so complex. The key is to follow the steps
+outlined here and `ask for help <https://www.astropy.org/help.html>`_ if you get stuck.
+The Astropy community is welcoming and friendly and will help you!
+
+This is divided into two sections, first a quickstart guide that provides an
```

Comment:
Should this paragraph also be moved under `{% if is_development %}`? Seems weird to explain there are two sections when in stable, it would ask you to go to latest anyway.

Reply:
Fixed.